helm_charts/obsrv-query-api/templates/configmap.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .Release.Name }}-config
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: "druid-proxy-api"
```

Comment:
can we follow one name ?

Reply:
Is this configurations are enough? how do you read the druid ip address details ?